### ISSUE TITLE
GraphStructure$ViewDestructorThread are never released Gephi Toolkit 0.8.7

### ISSUE THREAD

**Issue by elmer-garduno** [gephi/gephi#854]
As noted on http://forum.gephi.org/viewtopic.php?t=2242 the following threads are never killed when closing a workspace. This problem eventually leads to a memory leak.

Here is a partial thread dump:

```
"DHNS View Destructor" daemon prio=5 tid=7f93a7c4d800 nid=0x117b42000 in Object.wait() [117b41000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <7c16a9ae8> (a java.lang.Object)
	at java.lang.Object.wait(Object.java:485)
	at org.gephi.graph.dhns.core.GraphStructure$ViewDestructorThread.run(GraphStructure.java:240)
	- locked <7c16a9ae8> (a java.lang.Object)

"graph-event-bus" daemon prio=5 tid=7f93a733a000 nid=0x117a3f000 in Object.wait() [117a3e000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <7c16a78d8> (a java.lang.Object)
	at java.lang.Object.wait(Object.java:485)
	at org.gephi.graph.dhns.core.EventManager.run(EventManager.java:134)
	- locked <7c16a78d8> (a java.lang.Object)
	at java.lang.Thread.run(Thread.java:695)

"attribute-event-bus" daemon prio=5 tid=7f93a7db3000 nid=0x117736000 in Object.wait() [117735000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <7c16b7218> (a java.lang.Object)
	at java.lang.Object.wait(Object.java:485)
	at org.gephi.data.attributes.event.AttributeEventManager.run(AttributeEventManager.java:137)
	- locked <7c16b7218> (a java.lang.Object)
	at java.lang.Thread.run(Thread.java:695)
```

**Comment by sunqing** [comment]
Hello, Did you resolve this issue?
 I also met this problem. How do you deal with this problem.

**Comment by mbastian** [comment]
This has been resolved with Gephi 0.9+. Please try again with the latest toolkit release (0.9.1).
